src/OrchardCore.Modules/OrchardCore.AuditTrail/Settings/AuditTrailSettings.cs

```diff
@@ -0,0 +1,17 @@
+﻿using OrchardCore.AuditTrail.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OrchardCore.AuditTrail.Settings
+{
+    public class AuditTrailSettings
+    {
+        public bool EnableClientIpAddressLogging { get; set; }
+        public IEnumerable<AuditTrailEventSetting> EventSettings { get; set; } = 
+            Enumerable.Empty<AuditTrailEventSetting>();
+        public IEnumerable<BlacklistedContentTypesViewModel> BlacklistedContentTypes { get; set; } = 
```

Comment:
Reminder from the meeting:
Rename into IgnoredContentTypes or change the behaviour to set only the included ContentTypes.